package.json

```diff
@@ -7,7 +7,7 @@
     "build": "next build",
     "lint": "next lint",
     "start": "next start",
-    "test": "env TZ='UTC' jest",
+    "test": "set TZ=UTC && jest",
```

Comment:
what does this change do? we might have to update things on vercel to make it work on production

Reply:
I was missing the env dependencies. npm install -g win-node-env fixed this for me. undid this change.